lms/static/js/spec/utils/dropdown_spec.js

```diff
@@ -0,0 +1,122 @@
+                expect(menu.length).toBe(wrapper.length);
+                expect(menu_item.length).toBeGreaterThan(0);
+                expect(menu_action.length).toBe(menu_item.length);
+            });
+
+            it('ensures ARIA attributes are present on button and menu', function() {
+                expect(button).toHaveAttr({
+                    'aria-haspopup': 'true',
+                    'aria-expanded': 'false',
+                    'aria-controls': 'dropdown'
+                });
+
+                expect(menu).toHaveAttr({
+                    'tabindex': '-1'
+                });
+            });
+        });
+
+        describe('when running', function() {
+            function keyPressEvent(key) {
+                return $.Event('keydown', { keyCode: key });
+            }
+
+            var wrapper, button, menu, menu_item, menu_action, KEY = $.ui.keyCode,
+
+            it('opens the menu on button click', function() {
+                button.click();
+                expect(button).toHaveClass('is-active');
+                expect(button).toHaveAttr({
+                    'aria-expanded': 'true'
+                });
+                expect(menu).toHaveClass('is-visible');
+            });
+
+            it('closes the menu on outside click', function() {
+                $(window).click();
+                expect(button).not.toHaveClass('is-active');
+                expect(button).toHaveAttr({
+                    'aria-expanded': 'false'
+                });
+                expect(menu).not.toHaveClass('is-visible');
+                expect(menu).toHaveClass('is-hidden');
+            });
+
+            it('opens the menu on ENTER kepress', function() {
+                button.trigger(keyPressEvent(KEY.ENTER));
+                expect(button).toHaveClass('is-active');
+                expect(button).toHaveAttr({
+                    'aria-expanded': 'true'
+                });
+                expect(menu).toHaveClass('is-visible');
+                expect(menu.focus).toHaveBeenCalled();
+            });
+
+            it('opens the menu on DOWN keypress', function() {
+                button.trigger(keyPressEvent(KEY.DOWN));
+                expect(button).toHaveClass('is-active');
+                expect(button).toHaveAttr({
+                    'aria-expanded': 'true'
+                });
+                expect(menu).toHaveClass('is-visible');
+                expect(menu.focus).toHaveBeenCalled();
+            });
+
+            it('moves between menu items on UP or DOWN', function() {
+                var last_item = menu_item.length - 1, i;
+                container.trigger(keyPressEvent(KEY.ENTER));
+
+                for (i = last_item; i >= 0; i--) {
+                    menu_item.eq(i).trigger(keyPressEvent(KEY.UP));
+                }
+
+                for (i = 0; i <= last_item; i++) {
+                    menu_item.eq(i).trigger(keyPressEvent(KEY.DOWN));
```

Comment:
You aren't asserting anything in this test point. Make sure that the right item has the focus.